canopy_dispatch.ml

```diff
@@ -0,0 +1,108 @@
+open Lwt.Infix
+
+type store_ops = {
+  subkeys : string list -> string list list Lwt.t ;
+  value : string list -> string option Lwt.t ;
+  update : unit -> string list Lwt.t
+}
```

Comment:
That'll do for know, I have an idea on how to this but that's fine enough like this for the moment. :)